### PR TITLE
Update grants.md

### LINKED ISSUE #2395: Update information on W3F Grants Program
We've just merged the General and Open grants programs into a single program. The process is pretty much the same as it used to be for most applications, meaning: 

- Applications of up to $30k for first-time and $100k for subsequent grants are processed transparently on GitHub and paid out in crypto (BTC, DAI or USDT).
- Applications above those limits (no cap!), private applications and fiat payments still need to go through KYC and sign a contract.

Now this all happens on the same repo.

https://wiki.polkadot.network/docs/grants should be updated to reflect these changes.

Links can point to https://github.com/w3f/Grants-Program#web3-foundation-grants-program.

### PR DESCRIPTION
**Pull request by semuelle**
The "Open" and "General" grants programs have been merged into one. Also added links to some useful docs.

Resolves #2395.

### CODE REVIEW

**Comment by semuelle** [comment]
Pinging @Noc2.

File: docs/grants.md [hunk 1]
```diff
@@ -4,20 +4,18 @@ title: Web3 Foundation Grants
 sidebar_label: Grants
 ---
 
-The Web3 Foundation offers two types of grants:
+The Web3 Foundation offers grants for open source software development and research around Substrate, 
+Polkadot, Kusama and the Web3 ecosystem as a whole.
```
Comment (Noc2) [inline docs/grants.md]: I would remove "and the Web3 ecosystem as a whole"

**Comment by salmad3** [comment]
https://github.com/w3f/polkadot-wiki/pull/2404, we can rebase one and resolve merge conflicts or close one - which ever works.

File: docs/grants.md (hunk 1 above)
Comment (semuelle) [inline docs/grants.md]:
```suggestion
Polkadot, and Kusama. 
```

**Comment by semuelle** [comment]
> #2404, we can rebase one and resolve merge conflicts or close one - which ever works.

Sorry, I was not aware of the PR or the issue. I think this one provides a bit more info, but I'm not attached to it. As you like.

**Review by salmad3 — APPROVED** [review]
LGTM

**Pull request merged**